docs/nodle-wallets/nodle-cash/Rewards.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 4
+---
+
+# Nodle token and Network rewards
+
+### What is the NODL token?
+
+NODL is the native cryptocurrency of the Nodle network. It creates an incentive for participants like you to come and
+grow the network.
+
+### What is the NODL token price?
+
+You can check the price on [CoinMarketCap](https://coinmarketcap.com/currencies/nodleiot/) or just type in your browser 
+to get an answer.
+
+### How are rewards calculated and allocated?
+
+The amount of rewards you receive depends on your contribution to building the Nodle network and is calculated based on:
+1. **App time availability** - the percentage of time the Nodle app has been running on a phone and connected to the Internet. 
+3. **Bluetooth availability** - a metric indicating if the Bluetooth has been switched on on a phone. 
+4. **Geographical coverage** - your contribution to the network coverage in visited H3 tiles, which is calculated based
+on the number of nodes in each H3 tile.
+
+The total amount of rewards is set per day and is shared among all network nodes. It means that the amount 
+of rewards you receive will reduce over time as the network grows. However, the bigger the network, the higher the 
+NODL token price on the market. You can check the current token daily issuance [here](https://docs.nodle.com/docs/tokenomics/issuance#issuance-table)
```

Comment:
I feel like this should use a relative link to the page instead of the full form URL

Reply:
removed absolute URL link, placed relative link